types/lib/operators.d.ts

```diff
@@ -39,5 +40,6 @@ declare const Op: {
   readonly strictLeft: unique symbol;
   readonly strictRight: unique symbol;
   readonly substring: unique symbol;
+  readonly values: unique symbol;
```

Comment:
You will have to add this to `model.d.ts` too, probably with a signature of `(number | string)[]`.

Reply:
To be frank I'm not super comfortable editing `model.d.ts` with my limited knowledge of Sequelize, especially with `values` only being partially documented. I tried something, let me know if it makes sense.